app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ActiveRecord::Base
 
   attr_accessor :skip_password_validation
 
-  validates :email, presence: true
+  validates :email, presence: true, email_format: true
   validates :email, uniqueness: true, if: -> { email.present? }
```

Comment:
Can we move the `email_format` check to the `uniqueness` line? That way it will only validate if the email is actually present (And thus preventing double error messages :-) )